lib/lts/node-lts.json

```diff
@@ -25,8 +25,14 @@
   },
   "18.*": {
     "versionRange": ">=18.*",
-    "start_date": "2022-10-25T00:00:00.000Z",
+    "start_date": "2022-10-18T00:00:00.000Z",
```

Comment:
The releases page still says `2022-10-25` for the start date. any reason not to just keep it at `2022-10-25`?

Reply:
There is a 7 days period between node 16 entering maintenance and not active LTS(2022-10-18) and node 18 is not in active LTS according to the `start_date` on the release page. 
So to cover this difference we have to pull back the start_date. (node 18 is already LTS on
<img width="852" alt="Screen Shot 2022-10-19 at 3 01 46 PM" src="https://user-images.githubusercontent.com/12808214/196813207-96c1a65c-43e6-4fb3-b77c-f3b4f5b03cba.png">
 the release page)

Reply:
right, i'm suggesting addressing this gap by making the transition date later. so basically extend when 16 is supported rather than making folks move to 18 earlier.

Reply:
I moved it earlier because officially `18.x` is the current LTS, if we say `16.x`, won't it be a mismatch with what we have on the node LTS page?

Reply:
I mean the page conflicts itself, as you mentioned. The date does not match. I'm suggesting it's better to err on the side of requiring upgrading later rather than too early.